core/src/com/team3gdx/game/screen/GameScreen.java

```diff
@@ -677,6 +677,17 @@ public void checkInteraction(Cook ck, ShapeRenderer sr) {
 				stationManager.checkInteractedTile("", new Vector2(checkCellX, checkCellY));
 			}
 		}
+		Cell viewedTile2 = ((TiledMapTileLayer) map1.getLayers().get(0)).getCell(checkCellX, checkCellY);
+		if (viewedTile2 != null) {
```

Comment:
This will reduce performance for little benefit. An alternate approach would be an invisible tile that acts as an interactable 'overlay' to the graphical tile on the bottom layer.

Reply:
Fixed in 1e2e350dbc7b18d62a480870ab2c85349e7f87b9.